packages/core/content-releases/admin/src/pages/tests/Releases.test.tsx

```diff
@@ -0,0 +1,92 @@
+import { lightTheme, ThemeProvider } from '@strapi/design-system';
+import { render as renderRTL, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { IntlProvider } from 'react-intl';
+
+import { ReleasesPage } from '../Releases';
+
+const user = userEvent.setup();
+
+const render = () => ({
+  ...renderRTL(<ReleasesPage />, {
+    wrapper: ({ children }) => (
+      <ThemeProvider theme={lightTheme}>
+        <IntlProvider locale="en" messages={{}} defaultLocale="en">
+          {children}
+        </IntlProvider>
+      </ThemeProvider>
+    ),
+  }),
+});
+
+describe('Releases home page', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders the Releases title as an heading', () => {
+    const { getByRole } = render();
```

Comment:
applies to all the test file: please use the selectors via screen instead of destructuring the render return. It's something we decided in a frontend sync to have more consistency. It's also [recommended](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen) by the same RTL author